cli/providers/aws/translate.go

```diff
@@ -40,34 +42,89 @@ func Translate(s *service.Service, e *environment.Environment, config *aws.Confi
 		return "", fmt.Errorf("Region'%s' is not supported", e.Provider.Region)
 	}
 	t := gocf.NewTemplate()
-	err := loadASG(t, s, e, ami, config)
+	vpc, err := ec2.GetVPC(e, config)
 	if err != nil {
 		return "", err
 	}
-	err = loadELB(t, s, e)
+	domain := e.Provider.HostedZone[:len(e.Provider.HostedZone)-1]
```

Comment:
`strings.TrimSuffix(e.Provider.HostedZone, ".")`  does the same and I think it's more readable)